pycbc/strain/strain.py

```diff
@@ -278,6 +278,12 @@ def from_cli(opt, dyn_range_fac=1, precision='single',
         else:
             raise ValueError("Unrecognized precision {}".format(precision))
 
+        if opt.sample_rate:
+            logging.info("Resampling data")
+            strain = resample_to_delta_t(strain,
+                                         1.0 / opt.sample_rate,
+                                         method='ldas')
+
         if opt.gating_file is not None:
             logging.info("Gating times contained in gating file")
             gate_params = numpy.loadtxt(opt.gating_file)
```

Comment:
Should we not move this until *after* applying the gating file? Just that any sort of resampling will bleed a glitch out a bit, so it should be a little more powerful on the raw data ...